docs/redis_keys.md

```diff
@@ -39,14 +40,15 @@ The keys in the second category are the ones generated by _backend_ itself. Thes
-| `application/service_id:SERVICE_ID/id:APP_ID/keys`             | SERVICE_ID, APP_ID   | set    | `application/service_id:2/id:593a8c1a/keys`                        | `1) "91efe8a356c83e94b40e54cce211a1a4"`      | List of application keys associated to the application (for App Id / App Key mode) |
-| `application/service_id:SERVICE_ID/id:APP_ID/referrer_filters` | SERVICE_ID, APP_ID   | set    | `application/service_id:2/id:593a8c1a/referrer_filters`            | `1) "*.example.org" 2) "test.org"`           | List of referrer filters associated to the application                             |
+| Pattern                                                        | Parameters           | Type   | Example key                                                        | Example values                               | Description                                                                                      |
+|----------------------------------------------------------------|----------------------|--------|--------------------------------------------------------------------|----------------------------------------------|--------------------------------------------------------------------------------------------------|
+| `application/service_id:SERVICE_ID/id:APP_ID/plan_id`          | SERVICE_ID, APP_ID   | string | `application/service_id:2/id:593a8c1a/plan_id`                     | `"44"`                                       | Getting the plan ID for the application by its app ID and service ID                             |
+| `application/service_id:SERVICE_ID/id:APP_ID/plan_name`        | SERVICE_ID, APP_ID   | string | `application/service_id:2/id:593a8c1a/plan_name`                   | `"Test plan"`                                | Getting the plan name of the application by its app ID and service ID                            |
+| `application/service_id:SERVICE_ID/id:APP_ID/state`            | SERVICE_ID, APP_ID   | string | `application/service_id:2/id:593a8c1a/state`                       | `"active"`, or `"pending"`, or `"suspended"` | Getting the state of the application by its app ID and service ID                                |
+| `application/service_id:SERVICE_ID/id:APP_ID/keys`             | SERVICE_ID, APP_ID   | set    | `application/service_id:2/id:593a8c1a/keys`                        | `1) "91efe8a356c83e94b40e54cce211a1a4"`      | List of application keys associated to the application (for App Id / App Key mode)               |
+| `application/service_id:SERVICE_ID/id:APP_ID/redirect_url`     | SERVICE_ID, APP_ID   | string | `application/service_id:2/id:593a8c1a/redirect_url`                | `"https://example.com/redirect"`             | Redirect URL of the aplication (for application of services with OAuth/OpenID Connect auth mode) |
```

Comment:
I just added this line in the 2nd commit, the diff doesn't show it clearly because of reordering and fixing the columns widths.